### PR TITLE
feat: use ksyun rather than old arm cluster

### PR DESCRIPTION
**Pull request by lijie0123**
## Why:
- old arm k8s cluster can not allocate pod(debuging)
- add timeout
- add job dsl
## How:
- migrate from arm k8s cluster to default ksyun cluster
- add node selector for arm

### CODE REVIEW

File: jenkins/pipelines/cd/tiflow-operator-docker.groovy [hunk 1]
```diff
@@ -100,7 +100,8 @@ pipeline {
                     agent {
                         kubernetes {
                             yaml podYaml
-                            cloud "kubernetes-arm64"
+                            cloud "kubernetes"
+                            nodeSelector "kubernetes.io/arch=arm64"
```
Comment (wuhuizuo) [inline jenkins/pipelines/cd/tiflow-operator-docker.groovy:104]: Also need to set node selector for amd64 stage.

File: jenkins/jobs/cd/tiflow_docker.groovy [hunk 2]
```diff
@@ -0,0 +1,31 @@
+pipelineJob('tiflow-docker') {
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath('jenkins/pipelines/cd/tiflow-docker.groovy')
+            scm {
+                git{
+                    remote {
+                        url('git@github.com:PingCAP-QE/ci.git')
```
Comment (purelind) [inline jenkins/jobs/cd/tiflow_docker.groovy:9]: It is recommended to modify it to download via the https URL, as this repo is public.

Comment (purelind) [reply jenkins/jobs/cd/tiflow_docker.groovy:9]: https://github.com/PingCAP-QE/ci/blob/main/jobs/pingcap/tidb/latest/ghpr_build.groovy#L23-L35

File: jenkins/jobs/cd/tiproxy_docker.groovy [hunk 3]
```diff
@@ -0,0 +1,24 @@
+pipelineJob('tiproxy-docker') {
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath('jenkins/pipelines/cd/tiproxy-docker.groovy')
+            scm {
+                git{
+                    remote {
+                        url('git@github.com:PingCAP-QE/ci.git')
+                        credentials('github-sre-bot-ssh')
```
Comment (wuhuizuo) [inline jenkins/jobs/cd/tiproxy_docker.groovy:10]: Please use https url, and it's public, so no need credentials. Please check rest parts.

Comment (wuhuizuo) [reply jenkins/jobs/cd/tiproxy_docker.groovy:10]: ref: image

**Review by wuhuizuo — CHANGES_REQUESTED** [review]
please modify the job DSL files

**Review by wuhuizuo — APPROVED** [review]

**Pull request merged**